src/components/HotelCard.jsx

```diff
@@ -14,6 +15,19 @@ function HotelCard({
   availabilityFrom,
   availabilityTo,
 }) {
+  function formatUnixToNaturalDate(dateUnix) {
+    let dateNatural = new Date(dateUnix);
+    return `${dateNatural.getDate()}/${dateNatural.getMonth()}/${dateNatural.getFullYear()}`;
+  }
+
+  function dollars(number) {
```

Comment:
Es una buena practica prefixear las funciones con algun verbo que describa lo que hace, `dollars` resulta muy generica, en cambio `setDollar` o `printDollars` puede ser mejor. En general siempre se usa un set o get como prefijos, pero dependera de la empresa y sus patrones.
  
```suggestion
  function setDollars(number) {
```